test/test_torch.py

```diff
@@ -13716,6 +13716,15 @@ def test_float_scalar_pow_float_tensor(self, device):
         for base in floats:
             self._test_pow(base, tensor)
 
+    @onlyOnCPUAndCUDA
+    @unittest.skipIf(not TEST_NUMPY, 'Numpy not found')
+    @dtypes(*(torch.testing.get_all_dtypes(include_bool=False, include_bfloat16=False)))
+    def test_complex_scalar_pow_tensor(self, device, dtype):
+        complexes = [0.5j, 1. + 1.j, -1.5j, 2.2 - 1.6j]
+        tensor = torch.rand(100).to(dtype=dtype, device=device)
```

Comment:
For integer dtypes this resolves to a tensor full of zeros, which may not be the most interesting test case. We have a make_tensor function to generate a random tensor that would be nicer to use: https://github.com/pytorch/pytorch/blob/ecdbea77bca044ee97eb5f17d435d701e044b79f/torch/testing/_internal/common_utils.py#L1441-L1442

Reply:
Thanks so much for pointing out this issue, and offering the kind tip of using the `make_tensor` function.
#47101 has been created to address this comment and special cases of `zero exponents` and the `1 + 0j` base are added as well, please kindly help review.